Lang/Spanish.pj.Lang

```diff
@@ -177,7 +178,7 @@
 #320#  "Seleccionar directorio de ROMs actual"
 
 //Messages
-#340#  "¿ROM Dañado? Usa GoodN64 y verifica que la RDB esté actualizada."
+#340#  "¿ROM Dañada? Usa GoodN64 y comprueba por una RDB actualizada."

```

Comment:
It may not match word by word, but that is not important.
What is important is that the idea (up to date RDB check) is translated, and readable following best practices and common terms and style in the target language.

You can change this translation without needing to modify the English version.
Also, translated words can have very different contextual meanings.
If you are curious, compare "condescending" and "condescendiente". They are NOT perfectly equivalent.